src/RemoteInboxNotifications/GetRuleProcessor.php

```diff
@@ -57,6 +57,8 @@ public static function get_processor( $rule_type ) {
 				return new PluginVersionRuleProcessor(
 					new PluginsProvider()
 				);
+			case 'data':
```

Comment:
I initially didn't understand why this was called `data` - I see now that it allows for rules based on persistent data, not just what can be calculated at engine run time (like if there were no products at one point).

What do you think about naming this `state` or `store_state` instead? If you're good with a change, I think changing all of the `$data` variables makes sense as well.

Reply:
I'm going to go with `stored_state` if you think that's ok

Reply:
Done in dbad715d